index.html

```diff
@@ -90,12 +91,148 @@ <h1 class="tui__nav--logo tui__h-3">
       </nav>
     </header>
     <main class="lg2__main tui__m-xl">
+      <div class="lg2__carousel--wrapper">
+        <section class="tui__carousel">
+          <div class="tui__carousel--slides">
+            <input type="radio" name="radio-btn" id="radio1" />
+            <input type="radio" name="radio-btn" id="radio2" />
+            <input type="radio" name="radio-btn" id="radio3" />
+            <input type="radio" name="radio-btn" id="radio4" />
+
+            <div class="tui__carousel--slide tui__first">
+              <img
+                src="/assets/webps/books1.webp"
+                alt="books"
+                class="tui__img-hw100"
+              />
+            </div>
+
+            <div class="tui__carousel--slide">
+              <img
+                src="/assets/webps/serverroom.webp"
+                alt="server"
+                class="tui__img-hw100"
+              />
+            </div>
+
+            <div class="tui__carousel--slide">
+              <img
+                src="/assets/webps/book2.avif"
+                alt="books"
+                class="tui__img-hw100"
+              />
+            </div>
+
+            <div class="tui__carousel--slide">
+              <img
+                src="/assets/webps/book3.avif"
+                alt="books"
+                class="tui__img-hw100"
+              />
+            </div>
+
+            <div class="tui__carousel--manual-navigation">
+              <label for="radio1" class="tui__carousel--manual-btn"></label>
+              <label for="radio2" class="tui__carousel--manual-btn"></label>
+              <label for="radio3" class="tui__carousel--manual-btn"></label>
+              <label for="radio4" class="tui__carousel--manual-btn"></label>
+            </div>
+          </div>
+        </section>
+      </div>
       <h1 class="tui__h-1">Log<sub>2</sub></h1>
       <p class="tui__text-lg">
-        Log<sub>2</sub> is a store dedicated to selling books that fall around
-        computers and related stuff. Check out detailed list of products
-        <a href="/products">(here)</a>.
+        &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Any specific reason to use &nbsp; for spacing, you can use text-indent instead.

Reply:
With text-indent it is difficult to define the exact spacing for the next words to start. Especially with the responsive nature of fonts. You see the space char typically synchronizes with the rest of the font sizes and always starts at the right place where I am needing it to be.
There is one more way to achieve the same behavior that is if I fix these to text pieces together with the usage of relative and absolute positioning but again those pieces sound good but act differently. Moreover, I find it to be overkill to apply that. I even thought of implementing with grids but turned out that was also a lot hacky.
One more possible way was with the usage of ex but that too was too tricky in some sense (so I overlooked that one).
Apart from being simple I also found it to be a lot intuitive to use it this way. I believe I definitely need to read more literature on the same but as per the things I went through previously. It felt right to employ the spacing char in the same place.

Reply:
Didn't get where/how exactly you want to align this text. 
Try to get fix in next commit.

Reply:
Let's have a discussion today evening. We can surely work on this.

Reply:
Sure. Would like to work on it.